### PR TITLE
Improve printing of negative exponents in PolyElement

### PR DESCRIPTION
**Pull request by shivamvats**
cc @certik

### CODE REVIEW

**Comment by shivamvats** [comment]
@certik Could you have a look at this?

**Comment by certik** [comment]
I think this looks good, but you should also add regular tests for the negative exponents. You only have a few doctests.

File: sympy/polys/rings.py [hunk 1]
```diff
@@ -784,7 +784,7 @@ def str(self, printer, precedence, exp_pattern, mul_symbol):
                     continue
                 symbol = printer.parenthesize(symbols[i], prec_atom-1)
                 if exp != 1:
-                    if exp != int(exp):
+                    if exp != int(exp) or exp < 0:
```
Comment (certik) [inline sympy/polys/rings.py]: This change needs regular tests somewhere in `sympy/polys`.

**Comment by shivamvats** [comment]
@certik

**Comment by shivamvats** [comment]
@certik Could you have a look?

**Comment by certik** [comment]
I think this looks great, thanks for finishing it.

**Pull request merged**